server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/json/RequestObjectSerializationTest.scala

```diff
@@ -0,0 +1,237 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+
+package org.apache.james.jmap.json
+
+import java.net.URI
+
+import eu.timepit.refined.auto._
+import org.apache.james.jmap.model.CapabilityIdentifier
+//import org.apache.james.jmap.json.Invocation._
+//import org.apache.james.jmap.json.RequestObject._
+import org.apache.james.jmap.model.CreatedIds.{ClientId, ServerId}
+import org.apache.james.jmap.model.Id.Id
+import org.apache.james.jmap.model.Invocation.{Arguments, MethodCallId, MethodName}
+import org.apache.james.jmap.model.{CreatedIds, Invocation, RequestObject}
+import org.scalatestplus.play.PlaySpec
+import play.api.libs.json._
+
+class RequestObjectSerializationTest extends PlaySpec {
```

Comment:
We can use `WordSpec` like everywhere else and drop the `org.scalatestplus.play` library